x/group/simulation/genesis.go

```diff
@@ -0,0 +1,189 @@
+package simulation
+
+import (
+	"fmt"
+	"math/rand"
+
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	gogotypes "github.com/gogo/protobuf/types"
+
+	"github.com/regen-network/regen-ledger/x/group"
+)
+
+const (
+	GroupInfo        = "group-info"
+	GroupMembers     = "group-members"
+	GroupAccountInfo = "group-accout-info"
+	GroupProposals   = "group-proposals"
+	GroupVote        = "group-vote"
+)
+
+func getGroups(r *rand.Rand, accounts []simtypes.Account) []*group.GroupInfo {
+	groups := make([]*group.GroupInfo, 3)
+	for i := 0; i < 3; i++ {
+		acc, _ := simtypes.RandomAcc(r, accounts)
+		groups[i] = &group.GroupInfo{
+			GroupId:     uint64(i + 1),
+			Admin:       acc.Address.String(),
+			Metadata:    []byte(simtypes.RandStringOfLength(r, 10)),
+			Version:     1,
+			TotalWeight: "30",
+		}
+	}
+	return groups
+}
+
+func getGroupMembers(r *rand.Rand, accounts []simtypes.Account) []*group.GroupMember {
+	groupMembers := make([]*group.GroupMember, 3)
+	for i := 0; i < 3; i++ {
+		acc, _ := simtypes.RandomAcc(r, accounts)
+		groupMembers[i] = &group.GroupMember{
+			GroupId: uint64(i + 1),
+			Member: &group.Member{
+				Address:  acc.Address.String(),
+				Weight:   "10",
+				Metadata: []byte(simtypes.RandStringOfLength(r, 10)),
+			},
+		}
+	}
+	return groupMembers
+}
+
+func getGroupAccounts(r *rand.Rand, simState *module.SimulationState) []*group.GroupAccountInfo {
+	groupMembers := make([]*group.GroupAccountInfo, 3)
+	for i := 0; i < 3; i++ {
+		acc, _ := simtypes.RandomAcc(r, simState.Accounts)
+		any, err := codectypes.NewAnyWithValue(group.NewThresholdDecisionPolicy("10", gogotypes.Duration{Seconds: 1}))
+		if err != nil {
+			panic(err)
+		}
+		groupMembers[i] = &group.GroupAccountInfo{
+			GroupId:        uint64(i + 1),
+			Admin:          acc.Address.String(),
+			Address:        acc.Address.String(),
+			Version:        1,
+			DecisionPolicy: any,
+			Metadata:       []byte(simtypes.RandStringOfLength(r, 10)),
+		}
+	}
+	return groupMembers
+}
+
+func getProposals(r *rand.Rand, simState *module.SimulationState) []*group.Proposal {
+	proposals := make([]*group.Proposal, 3)
+	for i := 0; i < 3; i++ {
+
+		from, _ := simtypes.RandomAcc(r, simState.Accounts)
+		to, _ := simtypes.RandomAcc(r, simState.Accounts)
+
+		proposal := &group.Proposal{
+			ProposalId:          uint64(i + 1),
+			Proposers:           []string{simState.Accounts[0].Address.String(), simState.Accounts[1].Address.String()},
```

Comment:
move this `simState.Accounts[0].Address.String()` and `simState.Accounts[1].Address.String()` out of loop.